src/Main.java

```diff
@@ -1,6 +1,60 @@
-public class Main {
-    public static void main(String[] args) {
-        System.out.println("Задача №1");
+import java.time.LocalDate;
 
+ public class Main {
+
+     public static void main(String[] args) {
+         System.out.println("Задача №1");
+         printInfoAboutYear(2023);
+         System.out.println(" ");
+         System.out.println("Задача №2");
+         printInfoAboutVersion(0, 2023);
+         int DeliveryDays = calculateDeliveryDays(95);
```

Comment:
```suggestion
         int deliveryDays = calculateDeliveryDays(95);
```
названия переменных пишем ВСЕГДА с маленькой буквы